src/v2i-hub/PedestrianPlugin/src/FLIRWebSockAsyncClnSession.cpp

```diff
@@ -42,7 +76,7 @@ namespace PedestrianPlugin
          beast::error_code ec,
          tcp::resolver::results_type results)
     {
-        PLOG(logDEBUG) << "In FLIRWebSockAsyncClnSession::on_resolve " << std::endl;
+        PLOG(logDEBUG) << "In FLIRWebSockAsyncClnSession::on_resolve ";
```

Comment:
Can we delete this log statement. Not a big fan of log statement that just mark where you are in the code. Alternatively providing some information about current state would be helpful

Reply:
Agreed. Deleting all of them